### PR TITLE
Update links and remove Privacy Impact Assessment

### PR DESCRIPTION
**Pull request by janelx**
## Description of changes

Update links and removed direct mentions of Privacy Impact Assessment

## Reviewers:
- [ ] @MelissaBraxton : please review changes
- [x] @bpdesigns : please review changes

### CODE REVIEW

File: _pages/research/privacy.md [hunk 1]
```diff
@@ -23,30 +23,27 @@ subnav:
...
+- Ask key stakeholders to introduce you before conducting interviews with their team. [Email template: Stakeholder introduces researcher]({{ site.baseurl }}/resources/email-templates/stakeholder-introduces-researcher)
+- Build rapport with research participants in advance of the research session — for example, by emailing participants to see if they have any questions about the research, or briefly meeting with participants before the session begins to go over any logistical requirements. [Email template: Researcher sends agreement to a participant]({{ site.baseurl }}/resources/email-templates/researcher-sends-agreement)
 - Whenever you collect PII, store it digitally on [GSA’s approved systems for PII](https://www.gsa.gov/reference/gsa-privacy-program/privacy-impact-assessments-pia) and actively remove access to PII from anyone not on your project team; and always place paper documents with PII in locked file cabinets
```
Comment (bpdesigns) [inline _pages/research/privacy.md:47]: the link for storing PII should be the webpage on [SORNs](https://www.gsa.gov/reference/gsa-privacy-program/systems-of-records-privacy-act/system-of-records-notices-sorns-privacy-act) not PIA

**Review by bpdesigns — CHANGES_REQUESTED** [review]
Please see the inline comment around line 50

**Comment by bpdesigns** [comment]
Hi @janelx I added one comment that needs to be updated before approval.

File: _pages/research/privacy.md [hunk 2]
```diff
@@ -23,30 +23,27 @@ subnav:
...
 Each piece of collected or stored PII increases the risk of privacy violations. When possible, it’s best to refrain from collecting or storing PII at all (if you or your agency partner have questions about storing PII on third-party systems that have not been approved by GSA, you should consult with your partner agency’s Privacy Office). One way to do this is analyzing data collected via the [Digital Analytics Program](https://digital.gov/dap/).
 
-[The Privacy Act of 1974 [Justice.gov]](https://www.justice.gov/opcl/privacy-act-1974) provides protection to individuals by ensuring that Federal agencies:
+[The Privacy Act of 1974](https://www.justice.gov/opcl/privacy-act-1974) provides protection to individuals by ensuring that Federal agencies:
 
 {:.list-item--margin-bottom-extra}
 - Collect PII only when it is both legally authorized and necessary
 - Present [Privacy Act Notices]({{ site.baseurl }}/resources/privacy-act-notice) whenever they collect PII (in order to allow for informed consent)
 - Protect agency-held PII against anticipated threats to security or integrity which could result in substantial harm, embarrassment, inconvenience, or unfairness to the participant.
```
Comment (bpdesigns) [inline _pages/research/privacy.md:37]: In place of the PIA, please add 

"18F complies with the Privacy Act by following the information practices outlined in our [Privacy Act Statement for Design Research](https://www.gsa.gov/reference/gsa-privacy-program/privacy-act-statement-for-design-research on GSA.gov)."

**Comment by bpdesigns** [comment]
Hi @janelx in chatting with @MelissaBraxton we noted the opportunity to call out the Privacy Act Statement for Design Research

File: _pages/research/privacy.md (hunk 2 above)
Comment (janelx) [reply _pages/research/privacy.md:37]: @bpdesigns Which sentence would you like to replace? It this one: "_The following guidelines, drawn from our Privacy Impact Assessment for Design Research, help us build trust and protect privacy. This list isn’t exhaustive, but it’s a good place to start:"_?

Comment (bpdesigns) [reply _pages/research/privacy.md:37]: Apologies. I think I was unclear. Please add this sentence into the content you have. It replaces what was removed from lines 38-40 below.

**Comment by janelx** [comment]
@bpdesigns I believe I made all the requested updates and it's ready for another review. Thank you!

**Review by bpdesigns — APPROVED** [review]

**Pull request merged**